api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.7.x.xml

```diff
@@ -100,4 +100,88 @@
         </sql>
     </changeSet>
 	
+	<changeSet author="Isaiah (generated)" id="202408021920-1">
+		<preConditions onError="HALT" onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="concept_reference_range"/>
+			</not>
+		</preConditions>
+		<createTable tableName="concept_reference_range" >
+			<column name="concept_reference_range_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="concept_id" type="INT">
+				<constraints nullable="false"/>
+			</column>
+			<column name="concept_numeric_id" type="INT">
+				<constraints nullable="false"/>
+			</column>
+			<column name="criteria" type="VARCHAR(255)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="hi_absolute" type="DOUBLE"/>
+			<column name="hi_critical" type="DOUBLE"/>
+			<column name="hi_normal" type="DOUBLE"/>
+			<column name="low_absolute" type="DOUBLE"/>
+			<column name="low_critical" type="DOUBLE"/>
+			<column name="low_normal" type="DOUBLE"/>
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false" unique="true" />
+			</column>
+		</createTable>
+
+		<!-- foreign key constraints -->
+
+		<addForeignKeyConstraint
+			baseTableName="concept_reference_range"
+			baseColumnNames="concept_id"
+			constraintName="fk_concept_numeric_reference_range"
+			referencedTableName="concept_numeric"
+			referencedColumnNames="concept_id"/>
+
+	</changeSet>
+
+	<changeSet author="Isaiah (generated)" id="202408061820-1">
+		<preConditions onError="HALT" onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="obs_reference_range"/>
+			</not>
+		</preConditions>
+		<createTable tableName="obs_reference_range" >
+			<column name="obs_reference_range_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="obs_id" type="INT">
+				<constraints nullable="false"/>
+			</column>
+			<column name="hi_absolute" type="DOUBLE"/>
+			<column name="hi_critical" type="DOUBLE"/>
+			<column name="hi_normal" type="DOUBLE"/>
+			<column name="low_absolute" type="DOUBLE"/>
+			<column name="low_critical" type="DOUBLE"/>
+			<column name="low_normal" type="DOUBLE"/>
+			<column name="creator" type="INT"/>
+			<column name="date_created" type="TIMESTAMP"/>
+			<column name="voided" type="BOOLEAN"/>
```

Comment:
I think we do not need these fields because this object is a child whose parent already has them.

Reply:
mmh, this part I don't really understand. I know we want to make obs reference range embedable which makes sense to remove these fields. But support it was an entity which extends Voiable and creatable, would we still remove these fields? Then there wouldn't be need to extend these interfaces

Reply:
Yes do not extend those interfaces.